core/src/main/java/hudson/ClassicPluginStrategy.java

```diff
@@ -137,6 +137,7 @@ private static Manifest loadLinkedManifest(File archive) throws IOException {
                 } catch (InvalidPathException e) {
                     throw new IOException(e);
                 }
+                //noinspection StatementWithEmptyBody
```

Comment:
I am not sure we really want to have such instruction in the code. That's very specific to IntelliJ, compared to common annotation like `@SuppressWarning`. 
Opening the door for comment/annotation specific to all the popular tools is perhaps really desired.

For the moment we only have 2 of them, from you, in the core.

Reply:
I'll drop from this PR…

Reply:
FWIW I think an empty block with a comment in it is a legitimate tool to improve code readability and any tool should be configurable to ignore that with such a check. Perhaps a feature request to IntelliJ, or wherever this is from?

Reply:
This is from IntelliJ:
![image](https://user-images.githubusercontent.com/2119212/57708559-8baa8d80-7637-11e9-9e2d-f8e7f5c02238.png)
![image](https://user-images.githubusercontent.com/2119212/57708584-95cc8c00-7637-11e9-9490-fb8c6d788d23.png)

I think based on the comment that there isn't a current standard `@Suppress` annotation which is why IntelliJ is using a comment, in general it uses `@...` things...

Reply:
So, the problem is that the `@SuppressWarnings` are a function of the compiler (of which there are a couple), and there isn't one for this thing, so it seems that IDEs are using non-standard extensions.